bigquery_etl/cli/query.py

```diff
@@ -0,0 +1,144 @@
+"""bigquery-etl CLI query command."""
+
+import click
+import os
+from pathlib import Path
+import re
+import sys
+import string
+
+from bigquery_etl.metadata.parse_metadata import Metadata
+
```

Comment:
This should use local imports, and include `reformat` for use when writing sql:
```suggestion
from ..metadata.parse_metadata import Metadata
from ..format_sql.formatter import reformat

```